runway/commands/modules_command.py

```diff
@@ -421,13 +424,16 @@ def run(self, deployments=None, command='plan'):  # noqa pylint: disable=too-man
                                     region)
                         LOGGER.info("Module options: %s", module_opts)
                         with change_dir(module_root):
-                            getattr(
-                                determine_module_class(module_root, module_opts)(  # noqa
-                                    context=context,
-                                    path=module_root,
-                                    options=module_opts
-                                ),
-                                command)()
+                            module_class = determine_module_class(module_root,
+                                                                  module_opts['class_path'])
```

Comment:
@rgitzel sorry, one more -- please change `module_opts['class_path']` to `module_opts.get('class_path')`

Reply:
Good point. Bad habit.